### PR TITLE
Add topics to problem 'nth-prime'

### LINKED ISSUE #871: nth-prime: add topics
As part of the update to the exercism website, we want to add topics to each exercise. This should be topics that will most likely be relevant to the exercise even though you might be able to solve it without encountering that topic. There is a [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) used across all exercism tracks but you can add track specific ones as well if needed.

See issue #523 for a discussion about adding topics. There are also some exercises (e.g. hello-world and two-fer) that already have topics so you can look at them as examples. The topics need to be added to the entry for the relevant exercise in `config.json`.

Please comment on this issue or reference it in your pull request (using #issue-number) if you're going to be working on it. If you put "fixes #issue-number" in the pull request description then it will also automatically close this issue when the pull request gets merged :)

### PR DESCRIPTION
**Pull request by mhdsmr2**
Fixes #871 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -288,7 +288,13 @@
       "core": false,
       "difficulty": 4,
       "slug": "nth-prime",
-      "topics": null,
+      "topics": [
+      	"arrays",
+      	"exception_handling",
+      	"lists",
+      	"loops",
+      	"integers"
```
Comment (FridaTveit) [inline config.json]: This looks great! The only tiny problem left is that this should be indented two spaces in from `topics`. Currently it's only one space in :)

Comment (mhdsmr2) [reply config.json]: Okay.

Comment (mhdsmr2) [reply config.json]: I correct the indentation.

Comment (FridaTveit) [reply config.json]: I'm sorry, but it's still only one space in. The second commit you made has it with two spaces, but then the third commit changed it to only one space again. Do you think you could change it to two spaces? Sorry for being so picky, if you don't want to do it then I can merge it and fix it later :)

Comment (mhdsmr2) [reply config.json]: Did as you said. Please merge this now.

**Comment by mhdsmr2** [comment]
Did the changes as @FridaTveit said.

File: config.json (hunk 1 above)
Comment (FridaTveit) [reply config.json]: Thanks for doing that, I greatly appreciate it :) However, like I've said before, you don't have to tell me to merge things. I will review new changes as soon as I have time and I'll merge them when they're ready :)

**Comment by mhdsmr2** [comment]
Thank you. :)

**Pull request merged**